logic/resolvable/Unifier.java

```diff
@@ -117,10 +117,12 @@ private Unifier(Map<Retrievable, Set<Variable>> unifier, Requirements.Constraint
     public Map<Retrievable, Set<Variable>> mapping() {
         return unifier;
     }
+    public Map<Variable, Set<Retrievable>> inverseMapping(){ return reverseUnifier; }
```

Comment:
oh and also this should match the name of the var its returning, so lets call it `reverseUnifier`